packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -0,0 +1,262 @@
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import * as KeyCode from 'keycode-js';
+import classNames from 'classnames/bind';
+import FocusTrap from 'focus-trap-react';
+import { injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import ThemeContext from 'terra-theme-context';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
+
+import ColumnContext from '../utils/ColumnContext';
+import GridContext, { GridConstants } from '../utils/GridContext';
+import styles from './Cell.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * @private
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   */
+  intl: PropTypes.shape({ formatMessage: PropTypes.func }).isRequired,
+
+  /**
+   * String identifier of the row in which the Cell will be rendered.
+   */
+  rowId: PropTypes.string.isRequired,
+
+  /**
+   * String identifier of the column in which the Cell will be rendered.
+   */
+  columnId: PropTypes.string.isRequired,
+
+  /**
+   * The cell's row position in the table. This is zero based.
+   */
+  rowIndex: PropTypes.number,
+
+  /**
+   * The cell's column position in the table. This is zero based.
+   */
+  columnIndex: PropTypes.number,
+
+  /**
+   * Content that will be rendered within the Cell.
+   */
+  children: PropTypes.node,
+
+  /**
+   *  Boolean indicating if cell contents are masked.
+   */
+  isMasked: PropTypes.bool,
+
+  /**
+   * Boolean value indicating whether or not the column header is selectable.
+   */
+  isSelectable: PropTypes.bool,
+
+  /**
+   * Boolean indicating whether the Cell is currently selected.
+   */
+  isSelected: PropTypes.bool,
```

Comment:
Not needed

Reply:
We need this prop for grid support.  The subcomponents are reused.